modules/apps/29-fee/client/cli/tx.go

```diff
@@ -101,10 +101,10 @@ func NewPayPacketFeeAsyncTxCmd() *cobra.Command {
 // NewRegisterCounterpartyAddress returns the command to create a MsgRegisterCounterpartyAddress
 func NewRegisterCounterpartyAddress() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:     "register-counterparty [address] [counterparty-address] [channel-id] [port-id]",
+		Use:     "register-counterparty [address] [counterparty-address] [port-id] [channel-id]",
```

Comment:
Should we maybe start with `port-id` & `channel-id` instead of the addresses?

Reply:
Good point... maybe yeah.

Reply:
Maybe let's start indeed with `port-id` and `channel-id`. That's the order we have for transfer: `simd tx ibc-transfer transfer [src-port] [src-channel] [receiver] [amount]`.